PRIVACY_POLICY.md

```diff
@@ -0,0 +1,65 @@
+# Swarm-Extension Privacy Policy
+
+The Swarm Extension belongs to the Swarm Foundation with the purpose of offering web3 environment
```

Comment:
belongs? not sure what that implies legally, but maybe change to "was developed"?